Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1201,6 +1201,44 @@ As per `k8s Service <https://kubernetes.io/docs/concepts/services-networking/ser
 Cilium's eBPF kube-proxy replacement by default disallows access to a ClusterIP service from outside the cluster.
 This can be allowed by setting ``bpf.lbExternalClusterIP=true``.
 
+
+Troubleshooting
```

Comment:
@brb I think we also need to add a note about incompatibility between cgroup v1 and v2 to the installation step?